sql/moz-fx-data-shared-prod/telemetry_derived/adm_engagements_daily_v1/query.sql

```diff
@@ -103,7 +103,7 @@ WITH topsites_temp AS (
     END
     AS phase
   FROM
-    mozdata.telemetry.events
+    `moz-fx-data-shared-prod.telemetry.events`
```

Comment:
Unqualified is preferred https://github.com/mozilla/bigquery-etl#queries
```suggestion
    telemetry.events
```